projectglow/glow
# Add ganglia image / layer deployment to Glow docker

## What changes are proposed in this pull request?
Added `ganglia` image / layer deployment to Glow docker to support ganglia metrics collection and reporting in Databricks Container Services

## How is this patch tested?

Added an image / layer with a tested `ganglia` deployment and configuration designed specifically to perform metrics collection across the entire Spark cluster in Databricks Container Services. This implementation is in contained in the `./docker/databricks/dbr/dbr9.1/ganglia` folder. Note that the `ganglia` layer must be built last. 

Updated the .`/docker/databricks/build.sh` script to:
1) build the `projectglow/databricks-glow` image with the ganglia layer
2) build a new `projectglow/databricks-glow-minus-ganglia` image without the ganglia layer (for debugging and testing)

Updated the `README.md `file to reflect these changes and the rationale behind them
